sonar-text-plugin/src/test/java/org/sonar/plugins/secrets/api/SecretMatcherAssert.java

```diff
@@ -47,6 +47,7 @@ public static SecretMatcherAssert assertThat(SecretMatcher actual) {
   public SecretMatcherAssert behavesLike(SecretMatcher expectedMatcher) {
     isNotNull();
     usingRecursiveComparison()
+      .ignoringFields("preFilter")
```

Comment:
We shouldn't simply ignore it, but write test cases for it aswell.

Reply:
I'm not sure if it makes a lot of sense to also test predicates' behavior here since we already have tests for the predicate building.